packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/AddWalletButton/index.tsx

```diff
@@ -13,32 +14,32 @@ interface Props {
     device: TrezorDevice;
     instances: AcquiredDevice[];
     addDeviceInstance: (instance: TrezorDevice) => Promise<void>;
+    selectDeviceInstance: (instance: TrezorDevice) => Promise<void>;
 }
 
-const AddWalletButton = ({ device, instances, addDeviceInstance }: Props) => {
+const AddWalletButton = ({ device, instances, addDeviceInstance, selectDeviceInstance }: Props) => {
     const hasAtLeastOneWallet = instances.find(d => d.state);
-
-    let tooltipMsg: string | JSX.Element = '';
-    if (!hasAtLeastOneWallet) {
-        tooltipMsg = <Translation id="TR_TO_ACCESS_OTHER_WALLETS_DISCOVER" />;
-    }
-    if (!device.connected) {
-        tooltipMsg = <Translation id="TR_TO_ACCESS_OTHER_WALLETS" />;
-    }
-
+    const undiscoveredWallet = instances.find(d => !d.state);
```

Comment:
undiscoveredWallet is not necessary
you can do it like this:
`
onClick={() => hasAtLeastOneWallet ? addDeviceInstance(device) : selectDeviceInstance(instances[0])}
`